go/pkg/sysdb/grpc/server.go

```diff
@@ -158,7 +158,7 @@ func NewWithGrpcProvider(config Config, provider grpcutils.GrpcProvider) (*Serve
 		s.softDeleteCleaner.Start()
 
 		log.Info("Starting GRPC server")
-		s.grpcServer, err = provider.StartGrpcServer("coordinator", config.GrpcConfig, func(registrar grpc.ServiceRegistrar) {
+		s.grpcServer, err = provider.StartGrpcServer("sysdb-service", config.GrpcConfig, func(registrar grpc.ServiceRegistrar) {
```

Comment:
why this change? does anything downstream for telemetry use it are we sure nothing depends on it?

Reply:
the `coordinator` value was not previously used because `sysdb-service` was hard coded in the common helper